pkg/controller/releasemanager/incarnation.go

```diff
@@ -344,7 +357,7 @@ func (i *Incarnation) schedulePermitsRelease() bool {
 	}
 
 	// If engineers are out of office, don't start new releases
-	if !i.humaneReleasesEnabled && i.target().Release.Schedule != "always" {
+	if !i.picchuConfig.HumaneReleasesEnabled && i.target().Release.Schedule != "always" {
```

Comment:
Do you know where everything fails if picchuConfig is nil? Maybe we can make a check in the constructor just to avoid hard to debug regressions in the future?

Reply:
I made picchuConfig not a pointer.